tests/mir_test_doubles/stub_display_configuration.cpp

```diff
@@ -75,6 +76,7 @@ mtd::StubDisplayConfigurationOutput::StubDisplayConfigurationOutput(
     MirSubpixelArrangement subpixel_arrangement) :
         DisplayConfigurationOutput{
             id,
+            "OUT-" + std::to_string(id.as_value()),
```

Comment:
This code snippet appears in too many places.

Maybe an answer is to move the `name` member to the end of `DisplayConfigurationOutput` and provide an in-class default member initializer?